src/components/profile/Profile.js

```diff
@@ -20,22 +20,32 @@ import {
   StyledProfileUserInfoEditBtns,
   StyledProfileUserInfoFollowers,
 } from "./ProfileStyles"
-import { RiAddCircleLine } from "@react-icons/all-files/ri/RiAddCircleLine"
+import { RiAddCircleLine } from "react-icons/ri"
 
-import userImage from "../../assets/user-image.png"
-import post1 from "../../assets/post4.jpg"
-import post3 from "../../assets/post5.jpeg"
-import post2 from "../../assets/post6.jpeg"
+import userImage from "../../assets/avatars/user-image.png"
+import post1 from "../../assets/posts/post4.jpg"
+import post3 from "../../assets/posts/post5.jpeg"
+import post2 from "../../assets/posts/post6.jpeg"
 
 import { highlights } from "../../data/data"
 
+const userData = {
+  name: "Eisa Reazaei",
+  title: "front end web developer",
+  bio: "front end react web developer from iran country",
+  id: "eisa.rez",
+  followers: 675,
+  followings: 452,
+  posts: 6,
+  location: "Iran",
+}
 const Profile = () => {
   return (
     <Layout>
       <StyledProfilePageCt>
         <StyledProfileHeader>
           <span>
-            <p> eisa.rez</p>
+            <p> {userData.id}</p>
```

Comment:
space is extra, you better to handle spacings with margin or add `&nbsp;` to make white space